parse_export.py

```diff
@@ -114,30 +146,36 @@ def main():
                 object_count += len(parse_response['results'])
                 results['results'].extend(parse_response['results'])
             else:
-                parse_roundtrip_seconds = time.clock() - get_parse_data_startime
-                print ' retrieved: %.4f (in %.4f seconds)' % (object_count, parse_roundtrip_seconds)
                 break
 
         with open(os.path.join(temp_directory, '%s.json' % classname), 'w') as json_file:
             json_file.write(json.dumps(results, indent=4, separators=(',', ': ')))
 
-    print 'building archive...'
+        parse_roundtrip_seconds = time.time() - get_parse_data_startime
+
+        sys.stdout.write('  retrieved %d objects with %d reqs for %s in %.4f seconds \n' % (object_count, parse_request_count, classname, parse_roundtrip_seconds))
+        sys.stdout.flush()
+
+    sys.stdout.write('building archive... ')
+    sys.stdout.flush()
+
+    build_archive_starttime = time.time()
+
     with tarfile.open(name=archive_file_path, mode='w:bz2') as tar:
         with change_dir(temp_directory):
             for f in os.listdir('.'):
                 tar.add(f)
+    sys.stdout.write(' done. (in %.4f seconds)\n' % (time.time() - build_archive_starttime))
+    sys.stdout.flush()
```

Comment:
Hard to say. "print blah" was pretty pythonic back in the python 2 days.

Not sure what the kids like these days.

Not a big deal, it's just ugly.

On Mon, Sep 21, 2015 at 2:19 PM, Michael Hobbs <notifications@github.com>
wrote:

> In parse_export.py
> <https://github.com/expa/parse-export/pull/2#discussion_r40027489>:
>
> >      with tarfile.open(name=archive_file_path, mode='w:bz2') as tar:
> >          with change_dir(temp_directory):
> >              for f in os.listdir('.'):
> >                  tar.add(f)
> > +    sys.stdout.write(' done. (in %.4f seconds)\n' % (time.time() - build_archive_starttime))
> > +    sys.stdout.flush()
>
> what is the appropriate pattern?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/expa/parse-export/pull/2/files#r40027489>.
>